sessionctx/variable/session.go

```diff
@@ -976,6 +976,10 @@ type SessionVars struct {
 	// all the local data in each session, and finally report them to the remote
 	// regularly.
 	StmtStats *stmtstats.StatementStats
+	// StmtExecCtx represents the current statement execution information.
+	// Note that the information here is not updated in a timely，
+	// it is only updated at the time of `StmtStats.OnExecutionBegin` and `StmtStats.OnExecutionFinished`.
+	StmtExecCtx *stmtstats.StatementExecutionContext
```

Comment:
I think we can place it in `ExecStmt`, as a private field

Reply:
great, done.